### PR TITLE
Add ReadTheDocs documentation

### PR DESCRIPTION
**Pull request by extsoft**
#146

### CODE REVIEW

File: docs/md/concept.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Concept
+
+Modern build tools such as [Maven](https://maven.apache.org) or [Gradle](https://gradle.org) have
+a test phase of the build lifecycle to execute the unit tests of an application. This capability is
+used in the automated testing to manage the execution automated tests (instead of unit tests).
```
Comment (kermek) [inline docs/md/concept.md]: to manage the execution OF automated tests

File: docs/md/concept.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+The used solution has 3 main gaps:
+1. **There is no place for unit tests as it is already owned by automated tests.** But sometimes
+unit tests are useful even for the automated testing as they allow to check some specific
+implementations to be used during the automation.
+2. **Hard to configure tests suites.** This is because you need to run all unit tests at the same
+time during the project’s build cycle. And build tools support it very well. But for automated
+tests, it is often required to run only a subset of tests (or even only single test). The build
+tools are not so good at doing this.
+3. **There is no chance to create a JAR file and distribute automated tests like as standard
```
Comment (violant) [inline docs/md/concept.md]: like a

File: docs/md/entry-points.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+```
+You can wrap it with any additional logic which is required for your tests execution. For instance,
+some input arguments can be read to generate test data before testing, etc.
+
+If you need an example, please take a look for [jcat repository](https://github.com/extsoft/jcat/blob/master/src/main/java/pro/extsoft/comments/App.java).
+
+## Default TestNG entry point
+Class: [`org.tatools.sunshine.testng.Sunshine`](https://github.com/tatools/sunshine/blob/master/sunshine-testng/src/main/java/org/tatools/sunshine/testng/Sunshine.java)
+
+The class exposes the following behaviour:
```
Comment (violant) [inline docs/md/entry-points.md]: it could be this is typo "behavior"

Comment (extsoft) [reply docs/md/entry-points.md]: will fix

**Comment by extsoft** [comment]
@rultor please merge.

**Pull request merged**